utils/dataset_validator.py

```diff
@@ -0,0 +1,109 @@
+from flask_uuid import UUID_RE
+
+DATASET_NAME_LEN_MIN = 1
+DATASET_NAME_LEN_MAX = 100
+
+MIN_CLASSES = 2
+CLASS_NAME_LEN_MIN = 1
+CLASS_NAME_LEN_MAX = 100
+
+MIN_RECORDINGS_IN_CLASS = 2
+
+
+def validate(dataset, complete=False):
```

Comment:
what's complete? is this validation for building the model, instead of general validation?